user-guide/Advanced_Modules/Dashboards_app/Dashboard_Gateway_installation.md

```diff
@@ -67,3 +71,100 @@ There are two main reasons to consider a Dashboard Gateway setup:
 
    - *connectionString*: The hostname or IP address of the DataMiner Agent to which the Dashboard Gateway has to connect.
    - *connectionUser* and *connectionPassword*: The DataMiner user account that the Dashboard Gateway has to use to connect to the DataMiner Agent (username and password).
+
+## Reverse proxy
+
+Opening a dashboard or low-code app via the Gateway server will not work if they contain a web component displaying the [DataMiner Maps module](xref:maps) or [VideoThumbnails web page](xref:Linking_a_shape_to_a_video_thumbnail).
+
+A reverse proxy can be used to give access to these web pages on the DMA via the Gateway server. The reverse proxy will forward any web requests for these web pages to the DMA.
+
+### Reverse proxy requirements
+
+- IIS 7 or higher with ASP.NET role service enabled
+
+- [URL Rewrite](https://www.iis.net/downloads/microsoft/url-rewrite)
+
+- [Application Request Routing](https://www.iis.net/downloads/microsoft/application-request-routing)
+
+### Configuration
+
+#### Enabling reverse proxy functionality
+
+As the reverse proxy functionality is disabled by default, first enable it by doing the following:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a server node and select the *Application Request Routing* feature.
+
+1. Select the *Enable Proxy* checkbox.
+
+1. All the other settings can be left on default.
+
+#### Creating rewrite rules for the Maps module
+
+To add a rewrite rule that will proxy any request to the DataMiner Maps module at `https://mydma/maps/`, as long as the requested URL path starts with "maps", do the following:
+
+1. Open IIS Manager on the Gateway server.
+
+1. In the tree view on the left, choose a web site node and select the *URL Rewrite* feature.
```

Comment:
Is "web site" the correct term? Or should this be "website" or something else still?